trade-finance-manager-api/src/v1/__mocks__/mock-TFM-deal-AIN-submitted.js

```diff
@@ -0,0 +1,356 @@
+const MOCK_TFM_DEAL_AIN_SUBMITTED = {
+  _id: '61f94a2427c1a7009cde1b9d',
+  dealSnapshot: {
+    _id: { $oid: '61f94a2427c1a7009cde1b9d' },
+    additionalRefName: null,
+    bank: {
+      companiesHouseNo: 'UKEF0001',
+      emails: [
+        'maker1@ukexportfinance.gov.uk',
+        'checker1@ukexportfinance.gov.uk',
+      ],
+      id: '9',
+      mga: ['mga_ukef_1.docx', 'mga_ukef_2.docx'],
+      name: 'UKEF test bank (Delegated)',
+      partyUrn: '00318345',
+    },
+    bankInternalRefName: 'abc',
+    checkerId: '61f29adb6851c10012604bd1',
+    createdAt: 1643727396847,
+    dealType: 'GEF',
+    editedBy: ['61f29adb6851c10012604bce'],
+    comments: [
+      {
+        role: 'checker',
+        userName: 'BANK1_CHECKER1',
+        createdAt: 1643735218306,
+        comment: '123123',
+      },
+    ],
+    eligibility: {
+      criteria: [
+        {
```

Comment:
do we need all of these properties in here?

Reply:
Removed